src/script/components/list/ConversationListCell.tsx

```diff
@@ -157,17 +157,65 @@ const ConversationListCell = ({
     }
   }, [isFocused]);
 
-  const availabilityStrings: Record<string, string> = {
-    [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
-    [Availability.Type.AWAY]: t('userAvailabilityAway'),
-    [Availability.Type.BUSY]: t('userAvailabilityBusy'),
-  };
+  const availabilityStrings = useMemo(
+    () =>
+      new Map<Partial<Availability.Type>, string>([
+        [Availability.Type.AVAILABLE, t('userAvailabilityAvailable')],
+        [Availability.Type.AWAY, t('userAvailabilityAway')],
+        [Availability.Type.BUSY, t('userAvailabilityBusy')],
+      ]),
+    [],
+  );
```

Comment:
```suggestion  
  const availabilityStrings = useMemo(
    () => ({
      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
      [Availability.Type.AWAY]: t('userAvailabilityAway'),
      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
    } as const),
    [],
  );
```

The suggestion from @atomrc  is a bit different, not really qualified to comment tbh

Reply:
I change it to Map, with proper types :)